src/PlusDataCollection/StealthLink/vtkPlusStealthLinkTracker.cxx

```diff
@@ -656,18 +656,29 @@ PlusStatus vtkPlusStealthLinkTracker::GetImage(const std::string& requestedImage
     examImageDirectoryToDelete = examImageDirectory;
 
     vtkSmartPointer<vtkDICOMImageReader> reader = vtkSmartPointer<vtkDICOMImageReader>::New();

```

Comment:
Is there a particular reason why the reader instantiation is outside the try/catch block?

Reply:
The reader is still required later in the function, so I moved the declaration/instantiation outside the try/catch block.

Would it be better to have the declaration outside the try/catch, and move the instantiation back inside the block?